static/sass/print.scss

```diff
@@ -23,18 +23,21 @@ body {
   color: $color-x-dark;
   font-family: "Ubuntu", "Segoe UI", "Roboto", "Oxygen", "Cantarell",
     "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-  line-height: 1.3;
+  line-height: 1.5em;
```

Comment:
The line-height value in CSS is unitless - see: https://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/